css/training.css

```diff
@@ -105,7 +105,20 @@ h1:after {
     background-color: #73545A;
     color: #F6EDE4;
 }
-
+.main {
+    min-height: 100vh;
+    position: relative;/*←相対位置 */
+  }
+  footer {
+    position:absolute;
```

Comment:
What is this footer section? Is there any difference between main style.css and this training.css?